databricks_cli/pipelines/api.py

```diff
@@ -72,27 +124,30 @@ def _get_hashed_path(path):
             raise RuntimeError('Error \'{}\' while processing {}'.format(e, path))
 
         file_hash = hash_buffer.hexdigest()
-        path = 'dbfs:/pipelines/code/{}{}'.format(file_hash, os.path.splitext(path)[1])
+        path = '{}{}{}'.format(base_pipelines_dir, file_hash, os.path.splitext(path)[1])
         return path
 
-    def _get_remote_mappings(self, local_lib_objects):
-        return list(map(lambda llo: LibraryObject(llo.lib_type, self._get_hashed_path(llo.path)),
-                        local_lib_objects))
-
     def _get_files_to_upload(self, local_lib_objects, remote_lib_objects):
+        """
+        Returns a Local/Remote pair for every file that needs to be uploaded to dbfs
+        :param local_lib_objects: List[LibraryObject]
+        :param remote_lib_objects: List[LibraryObject]
+        :return: List[(LibraryObject, LibraryObject)]
+        """
         transformed_remote_lib_objects = map(
             lambda rlo: LibraryObject(rlo.lib_type, DbfsPath(rlo.path)),
             remote_lib_objects)
         return list(filter(lambda lo_tuple: not self.dbfs_client.file_exists(lo_tuple[1].path),
                            zip(local_lib_objects, transformed_remote_lib_objects)))
 
-    #
-    # Only required until the deploy/delete APIs requires the credentials in the body as well
-    # as the header.Once the API requirement is relaxed, this function can be stripped out and
-    # includes for this function removed.
-    #
     @staticmethod
     def _get_credentials_for_request():
+        """
+        Only required until the deploy/delete APIs requires the credentials in the body as well
+        as the header.Once the API requirement is relaxed, this function can be stripped out and
+        includes for this function removed.
```

Comment:
Only required while* and require*

Reply:
nit: `Once the API requirement is relaxed, this function can be stripped out includes for this function removed.` -> `Once the API requirement is relaxed, we can remove this function`.